dmriprep/interfaces/vectors.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""@authors: Derek Pisner, Oscar Estaban, Ariel Rokem"""
```

Comment:
no need for authorship tags, should be covered by the contributing guidelines (that's why I was so looking forward to passing them)